build-system/babel-plugins/babel-plugin-amp-mode-transformer/index.js

```diff
@@ -41,11 +41,14 @@ module.exports = function ({types: t}) {
         }
         specifiers.forEach((specifier) => {
           if (specifier.imported && specifier.imported.name === 'getMode') {
-            const filepath = resolve(
-              dirname(state.file.opts.filename),
-              source.value
+            const filepath = relative(
+              join(__dirname, '../../../'),
+              resolve(dirname(state.file.opts.filename), source.value)
             );
-            if (filepath.endsWith('/amphtml/src/mode')) {
```

Comment:
note: technically the parent folder does not need to be `amphtml`. I have multiple clones of the repo locally, only one named "amphtml". should be fixed with this change